packages/clay-multi-step-nav/docs/multi-step-nav.mdx

```diff
@@ -25,6 +28,16 @@ Each step can have two different states: `active` or `complete` defined by props
 
 <MultiStepNav />
 
+## State
+
+The current step can currently be represented by three states, `complete`, `active` or `error`.
+
+The active state can be set using the `active` property in the `ClayMultiStepNav.Item` or `ClayMultiStepNavWithBasicItems` component which defines the step index.
+
+The `complete` and `error` states are defined using the `state` property in the `ClayMultiStepNav.Item` which defines whether the item is complete or had an error in the current state, the `ClayMultiStepNavWithBasicItems` component follows the same API with the difference that it sets the state of the item to the current step which is set when using the `active` or `defaultActive` API.
+
+<MultiStepNavError />
+
 ## Indicator position
 
 Some use cases don't need the title or subtitle to add context to the steps, in those cases you might just want the label to be rendered to the `top` of the indicator, so it can truncate the text if it is long.
```

Comment:
The examples have already been updated, only the `<ClayMultiStepNav.Indicator />` we still maintain the `complete` API because we don't need to set any `error` state for this component just the `complete` state.

Reply:
It looks like Indicator would benefit from knowing about error state as well, so it's probably best that we update that api also to accept `state` instead of `complete`. A [this line](https://github.com/liferay/clay/blob/master/packages/clay-multi-step-nav/src/Indicator.tsx#L53) we should be checking if it's an error as well and only showing the label if `(!error && !complete)`. Right now that logic is split with the error check being [over here](https://github.com/liferay/clay/pull/5450/files#diff-ded261bc8c14e3687f770a378db7c7773320c8585ecc28c3a03823f17e33acccR157). It would be better if that was handled together otherwise you can end up with easy bugs like the image below if you're defining the `<ClayMultiStepNav.Indicator>` yourself.

![Screenshot 2023-04-05 at 9 51 14 AM](https://user-images.githubusercontent.com/1609196/229992685-10e5ebbb-c4d6-47ef-9d79-0eb50a668883.png)

Reply:
Yeah, I'm thinking of using the context to add just to `ClayMultiStepNav.Item` so I don't need to add this API to the Indicator and we can deprecate `complete` API.